src/cheri_sys_regs.sail

```diff
@@ -84,6 +85,7 @@ function legalize_ccsr(c : ccsr, v : xlenbits) -> ccsr = {
   let v = Mk_ccsr(v);
   /* For now these bits are not really supported so hardwired to true */
```

Comment:
This comment only applied to d/e FWIW and now only applies to e.  I wonder if we want a separate comment before the feature bits just so this older comment doesn't seem to apply to the feature bits?

Reply:
Should be fixed now.